Remora.Results/Result.cs

```diff
@@ -180,6 +182,19 @@ public static Result FromError<TEntity>(Result<TEntity> result)
 [PublicAPI]
 public readonly struct Result<TEntity> : IResult<TEntity>
 {
+    /// <summary>
+    /// Gets a successful result.
+    /// </summary>
+    public static Result<TEntity> Success => new(default, default, default);
```

Comment:
See previous comment. You can use the typed FromSuccess method as well.

Reply:
This would require suppressing a possible null ref warning as `Result<TEntity>.FromSuccess`' entity arg isn't marked as nullable.

Reply:
Hm, odd. I'll have to look into that. As it is, you can ignore that suggestion.